code/modules/martial_arts/krav_maga.dm

```diff
@@ -3,6 +3,17 @@
 	var/datum/action/neck_chop/neckchop = new/datum/action/neck_chop()
 	var/datum/action/leg_sweep/legsweep = new/datum/action/leg_sweep()
 	var/datum/action/lung_punch/lungpunch = new/datum/action/lung_punch()
+	var/datum/action/neutral_stance/neutral = new/datum/action/neutral_stance()
+
+/datum/action/neutral_stance
+	name = "Neutral Stance - You relax, cancelling your last Krav Maga stance attack."
+	button_icon_state = "neutralstance"
+
+/datum/action/neutral_stance/Trigger()
```

Comment:
It looks like this can be repeated infinitely, even if there is no active stance -- it could be used to spam the chat messages below.

Reply:
you can currently do this with all Krav maga actions. And I don't see it being abused, however I will restrict it to only if there is an active stance.